### ISSUE TITLE
Change Avatar to Buyer name on top right when Buyer is logged in

### ISSUE THREAD

**Issue by sanjay-goel** [alan345/nacho#181]
examples:
![image](https://user-images.githubusercontent.com/20733069/46919542-81d38580-cf95-11e8-8433-765071dbbff3.png)

![image](https://user-images.githubusercontent.com/20733069/46919551-a3347180-cf95-11e8-9ea0-86eb31d58723.png)

**Comment by sanjay-goel** [comment]
Yes, for now.  If we see Buyers uploading profile pictures we can reconsider it. Profile pics would be useful for building integration with social media, but in the early days it is probably not very useful.

**Comment by alan345** [comment]
![image](https://user-images.githubusercontent.com/15246526/47395503-d4224e00-d6db-11e8-999d-08ba0117338a.png)
